### DISCUSSION TITLE
Artifact update problem (npm): UNABLE_TO_GET_ISSUER_CERT_LOCALLY

### DISCUSSION THREAD

**Discussion by nl0** [renovatebot/renovate#12304] [Request Help]
Hey,
First of all, thanks for your app -- it's been a huge help in maintaining our npm dependencies fresh.
The problem is, it stopped working properly -- getting artifact update problems for every new PR:
```
npm ERR! code UNABLE_TO_GET_ISSUER_CERT_LOCALLY
npm ERR! errno UNABLE_TO_GET_ISSUER_CERT_LOCALLY
npm ERR! request to https://registry.npmjs.org/npm failed, reason: unable to get local issuer certificate
```

It worked in a fork of our main repo (https://github.com/akarve/quilt/) without any problems, but after installing it in our main repo (https://github.com/quiltdata/quilt), artifact update problems began (here's an example PR: https://github.com/quiltdata/quilt/pull/2406). Config hasn't been changed. Any ideas on what happened and how we can fix this?

**Comment by rarkins** [comment]
Duplicate of https://github.com/renovatebot/renovate/issues/12263

**Reply by nl0** [reply]
thanks for pointing that one out, i didn't manage to find it through search

**Comment by LyraHarring** [comment]
The “[Unable to Get Local Issuer Certificate](https://certera.com/kb/how-to-fix-ssl-certificate-problem-unable-to-get-local-issuer-certificate/)” error stems from a misconfigured SSL certificate on your local machine. This misconfiguration prevents a successful HTTPS handshake from taking place. The error is often linked to self-signed or corporate certificates, particularly with Git. (Self-signed certificates are mainly used in testing or development environments and are not verified by trusted Certificate Authorities. Git may throw an error when encountering such certificates. There are different ways to fix the error such as:-

1) Alter the php.ini File
2) Add SSL Certificate to Trusted Certificate Store
3) Reinstall Git & Select SSL Transport Backend Option
4) Grant Repository Access to SSL Certificates or Reassign Path in VS Code
5) Disable SSL certificate

May these steps will be helpful in resolving the issue.
